test/test-cases/typescript-css-modules/app/src/server.tsx

```diff
@@ -4,23 +4,34 @@ import { renderToString } from 'react-dom/server';
 import App from './App';
 
 interface SkuProps {
-  headTags: string;
-  bodyTags: string;
+  SkuProvider: React.FunctionComponent;
+  getHeadTags: () => string;
+  getBodyTags: () => string;
 }
-export default ({ headTags, bodyTags }: SkuProps) => ({
-  renderCallback: (_: any, res: any): void => {
+export default () => ({
+  renderCallback: (
+    { SkuProvider, getBodyTags, getHeadTags }: SkuProps,
+    _: any,
+    res: any
+  ): void => {
+    const app = renderToString(
+      <SkuProvider>
+        <App />
+      </SkuProvider>
+    );
+
     res.send(`
       <!DOCTYPE html>
       <html>
         <head>
           <meta charset="UTF-8">
           <title>My Awesome Project</title>
           <meta name="viewport" content="width=device-width, initial-scale=1">
-          ${headTags}
+          ${getHeadTags()}
```

Comment:
Why is this now a function call?

Reply:
Previously we created a list of assets for a build. The assets are now specific to each render so a function is required.